app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -84,39 +108,76 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     init {
         state = state.copy(isDataLoading = true)
         viewModelScope.launch {
-            when (val result = getUserInfo(userId)) {
-                is GetUserInfoResult.Failure -> {
-                    appLogger.d("Couldn't not find the user with provided id:$userId.id and domain:$userId.domain")
+            when (val conversationResult = getOrCreateOneToOneConversation(userId)) {
```

Comment:
This will actually create a conversation with every user whose profile we enter, should it work this way? Maybe we should have another use case `GetOneToOneConversation` (the one that doesn't create one if doesn't exist) and show the three dots button only if such conversation already exists?

Reply:
actually it creates `OneToOneConversation` only if it's not found. So I believe it's okay to use it here

Reply:
But it makes an unnecessary API request to create a conversation which probably also generates some socket messages when you just open someone's profile, which I think should be executed when you intentionally click and open the conversation 😉

Reply:
well, that data is needed for BottomSheet displaying. So we have 2 options here: 
1. leave it as it is
2. make that request only when user opens that BS, which may cause some delay in opening that BS